posts/inside-rust/2023-09-22-project-director-nominees.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Announcing the Project Director Nominees"
+author: Leadership Council
+team: Leadership Council <https://www.rust-lang.org/governance/teams/leadership-council>
+---
+
+# Announcing the Project Director Nominees
+
+Over the past couple weeks, the Rust Leadership Council has been soliciting nominees from the whole Rust project for the role of Project Director.
+Now that the nomination period is closed, we are happy to announce the list of nominees.
+As described in our [last post], we will elect from these people three new Project Directors.
+
+The nominees are:
+
+- Jakob Degen
+- Amanieu
+- Felix Klock
+- lcnr
+- scottmcm
+- Michael Goulet
+- Santiago Pastorino
+- Jubilee
+- Mara Bos

```

Comment:
it might make sense to make these links to e.g. GitHub or other notable profiles :)

Reply:
Agreed! Github is probably a good common denominator.

Going forward, I wonder if we should have some kind of project directory that can serve as a canonical place to link when we need to reference people?

Reply:
I just pushed an update that includes GH links for everyone.